### PR TITLE
[maistra-866] Remove OCP 3.11 references

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

File: topics/docs/installation/installation-options.adoc [hunk 1]
```diff
@@ -255,9 +255,15 @@ If you intend to use a custom image, you must override all three values of hub,
...
         template: all-in-one
+        elasticSearch:
+          nodeCount: 3
+          resources:
+            requests:
+              memory: "16Gi"
+              cpu: "1"
+            limits:
+              memory: "16Gi"
 ----
```
Comment (jwendell) [inline topics/docs/installation/installation-options.adoc]: This is covered in #89, no?

File: topics/docs/monitoring_and_tracing/kiali.adoc [hunk 2]
```diff
@@ -5,34 +5,106 @@ category: "Monitoring & Tracing"
...
 {{% /notice %}}
 
 == What is Kiali
 
-Kiali is a frontend for Maistra that will allow you to manage and monitor your mesh from a single UI. This UI will
-allow you to view configurations, monitor traffic flow and health, and analyze traces.
+
+link:http://kiali.io[Kiali] is a component which visualises an application's
+topology, health, metrics and tracing information to provide a deeper
+understanding of the application's behaviour and performance. Within a single
```
Comment (dmitri-d) [inline topics/docs/monitoring_and_tracing/kiali.adoc]: behavior vs. behaviour? We probably use US spelling everywhere else...

**Comment by jwendell** [comment]
Isn't it better to split this into two pr's? one for kiali and other one for ocp 3.11 removal?

File: topics/docs/monitoring_and_tracing/kiali.adoc [hunk 3]
```diff
@@ -5,34 +5,106 @@ category: "Monitoring & Tracing"
...
-```
-$ oc get route -n istio-system kiali -o jsonpath='{.spec.host}'
-kiali-istio-system.127.0.0.1.nip.io
-```
+Every pod deployed as part of the Service Mesh contains a proxy container
+responsible for intercepting the application traffic coming into and going
+out of the pod. The Service Mesh proxy is automatically created on deployment
+of the application and automatically captures information as part of each
+service invocation. Kiali uses this information to provide a visualisation of
+the Service Mesh behaviour. The information includes metrics as well as
```
Comment (dmitri-d) [inline topics/docs/monitoring_and_tracing/kiali.adoc]: ditto

File: topics/docs/monitoring_and_tracing/kiali.adoc (hunk 2 above)
Comment (brian-avery) [reply topics/docs/monitoring_and_tracing/kiali.adoc]: lol. For some reason my laptop has decided lately that anything not UK English is spelled incorrectly.

File: topics/docs/monitoring_and_tracing/kiali.adoc [hunk 4]
```diff
@@ -5,34 +5,106 @@ category: "Monitoring & Tracing"
...
+* Requests percentage
+* Last 1m
+* Every 10s
+
+image:kiali-graph-1.png[Kiali Graph showing configuration]
+
+You should now see the application graph showing traffic being created by our
+script generating load. The traffic will flow from the Service Mesh ingress
+gateway to the customer service, preference service and finally the
+recommendation service. The request percentage will show _100%_ on each edge
```
Comment (dmitri-d) [inline topics/docs/monitoring_and_tracing/kiali.adoc]: Should there be underscores around ```100%```?

File: topics/docs/monitoring_and_tracing/kiali.adoc [hunk 5]
```diff
@@ -5,34 +5,106 @@ category: "Monitoring & Tracing"
...
+except within the recommendation service where the traffic should be split
+evenly across all three versions.
+
+image:kiali-graph-2.png[Kiali Graph showing application traffic]
+
+=== Kiali's Distributed Tracing
+
+Within the Kiali UI select the _Distributed Tracing_ option from the left
```
Comment (dmitri-d) [inline topics/docs/monitoring_and_tracing/kiali.adoc]: *Distributed Tracing*?

Comment (dmitri-d) [reply topics/docs/monitoring_and_tracing/kiali.adoc]: Ugh, I meant replace "_"s with "*"s

Comment (dmitri-d) [reply topics/docs/monitoring_and_tracing/kiali.adoc]: ?

File: topics/docs/monitoring_and_tracing/kiali.adoc [hunk 6]
```diff
@@ -5,34 +5,106 @@ category: "Monitoring & Tracing"
...
+hand navigation and then choose
+
+* Namespace: istio-tutorial
+* Service: customer.istio-tutorial
+
+and finally press the _Search_ button.
```
Comment (dmitri-d) [inline topics/docs/monitoring_and_tracing/kiali.adoc]: Ditto

File: topics/docs/monitoring_and_tracing/kiali.adoc [hunk 7]
```diff
@@ -5,34 +5,106 @@ category: "Monitoring & Tracing"
...
+
+image:kiali-tracing-1.png[Kiali Tracing showing configuration]
+
+You should now see a list of the 20 most recent invocations of the customer service.
+
+image:kiali-tracing-2.png[Kiali Tracing showing invocations]
+
+Select one of the invocations to see more information. The graph should now
+show a hierarchy of the invocations with those spans highlighted in _red_
```
Comment (dmitri-d) [inline topics/docs/monitoring_and_tracing/kiali.adoc]: No undescores around "red"?

File: topics/docs/monitoring_and_tracing/kiali.adoc (hunk 2 above)
Comment (dmitri-d) [reply topics/docs/monitoring_and_tracing/kiali.adoc]: How peculiar.

**Review by dmitri-d — APPROVED** [review]

**Comment by brian-avery** [comment]
Moved the Kiali side of this to #95

**Review by jwendell — APPROVED** [review]

**Pull request closed without merging**